### PR TITLE
Fby cl init

### PR DESCRIPTION
**Pull request by joakim-hove**
My RPI is headless

### CODE REVIEW

**Comment by joakim-hove** [comment]
It works for me - I now have the controlpanel up and running; viewed on my laptop :+1:

File: fby_init [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    config = DeviceConfig.download( config_url )
+except KeyError:
+    locked_device( device_id )
+except ValueError:
+    wrong_id( device_id )
+    
+config.save( filename = config_path )
+
+fbi.manage("start")
+
```
Comment (pgdr) [inline fby_init]: please make this into a `__name__ == '__main__'` style file and put things into functions.

File: fby_init [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    
+if len(sys.argv) < 2:
+    error_exit( )
+
+device_id = sys.argv[1]
+
+
+root_url = settings.ROOT_URL
+sensor_path = settings.SENSOR_PATH
+config_url = root_url + "%s/%s/" % (sensor_path, device_id)
```
Comment (pgdr) [inline fby_init]: couldn't there be a function in `settings` that:

```python
@classmethod
def config_url(cls, sensor_path, device_id):
    return '%s%s/%s/' % (self.ROOT_URL, sensor_path, device_id)
```

Then you don't have to care about the concept of `root_url` outside whoever knows about the full `config_url`.

**Comment by joakim-hove** [comment]
OK - comments addressed.

File: tests/test_code.py [hunk 3]
```diff
@@ -36,9 +36,12 @@ def _get_lintable_files(self, folder):
     def test_linting(self):
         files = self._get_lintable_files('friskby_controlpanel')
         files = files + self._get_lintable_files('tests')
+        files = files + self._get_lintable_files('bin')
         for f in files:
             self.assertEqual(lint.lint(f), 0, 'linting required for %s' % f)
 
 
+
```
Comment (pgdr) [inline tests/test_code.py]: there's suddenly two new lines here, one with trailing whitespace.  Please run `M-x whitespace-cleanup` on this file, or use git tools to remove.

File: tests/test_code.py [hunk 4]
```diff
@@ -36,9 +36,12 @@ def _get_lintable_files(self, folder):
     def test_linting(self):
         files = self._get_lintable_files('friskby_controlpanel')
         files = files + self._get_lintable_files('tests')
+        files = files + self._get_lintable_files('bin')
```
Comment (pgdr) [inline tests/test_code.py]: I don't think just adding `bin` to lint folders is enough.  We only collect files ending with `*.py`.  I think it should be enough to also add files starting with `fby_`, i.e., in the walk, add `in fnmatch.filter('fby_*')`.

**Comment by joakim-hove** [comment]
OK.

**Pull request merged**